exercises/outcomes/GT/GT-ready/template.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<knowl mode="exercise" xmlns="https://spatext.clontz.org" version="0.2">
+    <intro><p>Choose the most appropriate response for each question.</p></intro>
+    <!-- {{#questions}} -->
+    <!-- {{#area_comp}} -->
+    <knowl>
+        <content>
+            <p>
+Find the area of the parallelogram with vertices
+<m>{{verts}}</m>.
+            </p>
+            <list>
+                <!-- {{#choices}} -->
+                <item><p>({{letter}}) <m>{{item}}</m></p></item>
+                <!-- {{/choices}} -->
+            </list>
+        </content>
+        <outtro>
+            <!-- {{#choices}} -->
+                <!-- {{#correct}} -->
+            <p>({{letter}}) <m>{{item}}</m></p>
+                <!-- {{/correct}} -->
+            <!-- {{/choices}} -->
+        </outtro>
+    </knowl>
+    <!-- {{/area_comp}} -->
+    <!-- {{#area_adj}} -->
+    <knowl>
+        <content>
+            <p>
+A parallelogram <m>P</m> has area <m>{{area}}</m>. If another
```

Comment:
I am more concerned about the lack of images with this one.  I wonder if we could scaffold this by giving them some coordinates (or vectors) for the vertices of P?  While not necessary, that might get some students to draw a picture.

Reply:
I thought about this but would need to be more careful about picking nice numbers and vertices. There's probably some Pick's Theorem trickery I could do here but I'd like to try this as-is first and see how it goes in a couple weeks.

Reply:
Not really, the pictures we had before mostly had the base along the x axis, so it was very easy to see base and height from the picture.  For these purposes, I'd think (0,0), (a,0), (b,c), (a+b,c) would be fine.

Reply:
These are the questions this adapts:

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/25fe272c-6384-4b83-ab96-8a55ff1abffe)

Reply:
Well, if you tilt your head... 🤪